controllers/tasks/replication/demote.go

```diff
@@ -0,0 +1,15 @@
+package replication
+
+type DemoteImageTask struct {
+	// params required for this task
+}
+
+func NewDemoteImageTask() *DemoteImageTask {
+	return &DemoteImageTask{}
+}
+
+func (t *DemoteImageTask) Run() error {
```

Comment:
```suggestion
func (d *DemoteImageTask) Run() error {
```

Reply:
Take care of it in other places also let's use the first letter from the struct name